content-src/asrouter/asrouter-content.jsx

```diff
@@ -319,6 +321,7 @@ export class ASRouterUISurface extends React.PureComponent {
     const {message, bundle} = this.state;
     if (!message.id && !bundle.template) { return null; }
     const shouldRenderBelowSearch = TEMPLATES_BELOW_SEARCH.includes(message.template);
+    const shouldRenderInHeader = TEMPLATES_ABOVE_PAGE.includes(message.template);
```

Comment:
Do we want to support showing the below-search snippet for about:welcome? I suppose generally, would we need to support showing multiple templates potentially in above page, below search, and in footer?

Reply:
aaron says about:welcome has its own special "snippet" anyway, so only one at a time. So I believe this approach should be okay where if trailhead is active, then nothing else